Dependencies/napi/napi-jsi/include/napi/js_native_api.h

```diff
@@ -0,0 +1,419 @@
+#ifndef SRC_JS_NATIVE_API_H_
```

Comment:
I'm not sure what is going on with this. We can't include the low-level C N-API files for napi-jsi which is an adapter for the C++ layer. Can we chat offline?